apps/preview/nuxt/pages/showcases/Search/SearchBasic.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <form @submit.prevent="submit" ref="referenceRef" class="relative">
+    <label>
+      <SfInput ref="inputRef" v-model="inputModel" @focus="open" placeholder="Search">
```

Comment:
```suggestion
      <SfInput ref="inputRef" v-model="inputModel" @focus="open" placeholder="Search 'MacBook' or 'iPhone'...">
```
Different placeholder than in react component